CRBM-Viz/src/app/app.module.ts

```diff
@@ -52,10 +60,31 @@ const declarations = [
 ];
 
 //Services
-import { VisualizationsService } from './Services/visualizations.service';
-import { SimulationService } from './Services/simulation.service';
-import { AuthService } from './Services/auth.service';
-const providers = [VisualizationsService, SimulationService, AuthService];
```

Comment:
Could you edit this portion to ensure that the visualization and simulation services are imported correctly as well?